src/api/requests/fetchFilingSubmissionLatest.ts

```diff
@@ -70,24 +70,28 @@ async function retryRequestWithDelay(
   // eslint-disable-next-line no-param-reassign
   axiosInstance.defaults.retryCount += One;
 
-  // TODO: Remove console.logs once retry adjustments have been fully accepted
-  console.log(
-    'Validation STILL in-progress - Long Polling - RETRYING',
-    response,
-  );
+  if (import.meta.env.DEV) {
```

Comment:
Wonder how the security scan stuff will handle these dev only console.logs. We'll find out. 👍